akka-serialization-jackson/src/test/scala/akka/serialization/jackson/JacksonSerializerSpec.scala

```diff
@@ -145,32 +145,54 @@ class JacksonJsonSerializerSpec extends JacksonSerializerSpec("jackson-json") {
   "JacksonJsonSerializer with Java message classes" must {
     import JavaTestMessages._
 
-    // see SerializationFeature.WRITE_DATES_AS_TIMESTAMPS
-    "by default serialize dates and durations as numeric timestamps" in {
+    // see SerializationFeature.WRITE_DATES_AS_TIMESTAMPS = off
+    "be possible to serialize dates and durations as text with ISO-8601 date format (default)" in {
+      // Default format is defined in com.fasterxml.jackson.databind.util.StdDateFormat
```

Comment:
IIUC, this PR makes the serialization into ISO-8601 the default. 
Then, aren't testnames in lines 149 and 171 backwards? (e.g. line 149 should state it tests the default)

Reply:
no, I think it's correct, maybe you got the WRITE_DATES_AS_TIMESTAMPS backwards?

`WRITE_DATES_AS_TIMESTAMPS = on` => numeric timestamps (array)
`WRITE_DATES_AS_TIMESTAMPS = off` => ISO-8601

Reply:
My original comment was too short and ambiguous. What I meant is:

 - `reference.conf`sets `FAIL_ON_UNKNOWN_PROPERTIES = off` so the default is `ISO-8601`
 - line 152 states `"be possible to serialize dates and durations as text with ISO-8601 date format (default)"`. I just noticed the `(default)` at the end of the sentence but I still think the test should read `"by default serialize dates and durations as numeric timestamps"`
 - line 175 states `"by default serialize dates and durations as numeric timestamps"` but in that test the overwrite `FAIL_ON_UNKNOWN_PROPERTIES = off` is necessary. 

I think the test names should be:

 - `151`: `"by default serialize dates and durations as text with ISO-8601 date format"` 
 - `175`: `"be possible to serialize dates and durations as numeric timestamps"`

Does that make sense?

Reply:
that makes perfect sense, you are right, fixed